### PR TITLE
[strings] Translation of Edit track and Choose color

### PR DESCRIPTION
**Pull request by kavikhalique**

### CODE REVIEW

File: android/app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -864,6 +864,8 @@
     <string name="track_recording_toast_nothing_to_save">La ruta está vacía - no hay nada que guardar</string>
     <!-- Error message when there are no File Manager apps installed to select a folder when importing Bookmarks and Tracks -->
     <string name="error_no_file_manager_app">No se puede mostrar el diálogo de selección de carpetas porque no hay ninguna app adecuada instalada en su dispositivo. Instale una aplicación de gestión de archivos e inténtelo de nuevo</string>
+  <string name="choose_color">Elige el color</string>
```
Comment (andergd) [inline android/app/src/main/res/values-es/strings.xml:867]: Elegir el color

File: android/app/src/main/res/values-eu/strings.xml [hunk 2]
```diff
@@ -859,6 +859,8 @@
     <string name="track_recording_toast_nothing_to_save">Ibilbidea hutsik dago; ez dago ezer gordetzeko</string>
     <!-- Error message when there are no File Manager apps installed to select a folder when importing Bookmarks and Tracks -->
     <string name="error_no_file_manager_app">Ezin da karpeta hautatzeko elkarrizketa-koadroa erakutsi zure gailuan ez dagoelako aplikazio egokirik instalatuta. Instalatu fitxategi-kudeatzaile aplikazio bat eta saiatu berriro</string>
+  <string name="choose_color">Aukeratu kolorea</string>
+  <string name="edit_track">Ibilbidea editatu</string>
```
Comment (andergd) [inline android/app/src/main/res/values-eu/strings.xml:863]: Editatu ibilbidea

File: android/app/src/main/res/values-eu/strings.xml [hunk 3]
```diff
@@ -859,6 +859,8 @@
...
     <string name="uri_open_location_failed">Ez dago kokapena ireki dezakeen aplikaziorik instalatu</string>
```
Comment (andergd) [inline android/app/src/main/res/values-eu/strings.xml:864]: Last word: instalatuta

**Comment by soshial** [comment]
Latvian (lv):
```xml
<string name="choose_color">Izvēlieties krāsu</string>
<string name="edit_track">Ceļa labošana</string>
```

**Comment by matheusgomesms** [comment]
PT:   <string name="edit_track">Editar trajeto</string>
________________________________
De: soshial ***@***.***>
Enviado: Thursday, October 17, 2024 10:06:57 PM
Para: organicmaps/organicmaps ***@***.***>
Cc: Matheus Gomes ***@***.***>; Review requested ***@***.***>
Assunto: Re: [organicmaps/organicmaps] [strings] Translation of Edit track and Choose color (PR #9529)


Latvian (lv):

<string name="choose_color">Izvēlieties krāsu</string>
<string name="edit_track">Ceļa labošana</string>

File: data/strings/strings.txt [hunk 4]
```diff
@@ -32344,10 +32344,92 @@
...
+    ko = 색상 선택
+    lt = Pasirinkite spalvą
+    mr = रंग निवडा
+    nb = Velg farge
+    nl = Kies een kleur
+    pl = Wybierz kolor
+    pt = Escolher cor
+    pt-BR = Escolher cor
+    ro = Alegeți culoarea
+    ru = Выберите цвет
```
Comment (pastk) [inline data/strings/strings.txt:32378]: ru is good!

Comment (soshial) [reply data/strings/strings.txt:32378]: I dont really like that this is in the infinitive form `<string name="edit_track">Редактировать трек</string>`. Why not make it more descriptive of the screen ant the current process: `<string name="edit_track">Редактирование трека</string>`

File: android/app/src/main/res/values-eu/strings.xml (hunk 3 above)
Comment (kavikhalique) [reply android/app/src/main/res/values-eu/strings.xml:864]: this change is not related to this pr should i still change it in strings.txt file??

**Comment by NeatNit** [comment]
What exactly does 'Edit Track' mean? What can be edited?

Anyway, I think Hebrew (he) for edit track should be: עריכת מסלול

**Comment by kavikhalique** [comment]
> What exactly does 'Edit Track' mean? What can be edited?
> 
> Anyway, I think Hebrew (he) for edit track should be: עריכת מסלול

Edit track will be shown as title when users will choose a track to edit its name, desc, color etc in bookmarks

**Comment by kavikhalique** [comment]
> Latvian (lv):
> 
> ```
> <string name="choose_color">Izvēlieties krāsu</string>
> <string name="edit_track">Ceļa labošana</string>
> ```

I am sorry but i cannot find the (lv) lang in strings.txt can you please tag the line or tell me the correct abbreviation used in strings.txt file.

**Comment by NeatNit** [comment]
> I am sorry but i cannot find the (lv) lang in strings.txt

I think you're supposed to add it

> Edit track will be shown as title when users will choose a track to edit its name, desc, color etc in bookmarks

Thanks for the answer, my suggested change still stands.

File: data/strings/strings.txt (hunk 4 above)
Comment (pastk) [reply data/strings/strings.txt:32378]: It just was like that for a long time and is used both for a button and a screen, e.g. we have "Редактировать метку" and this is analogous.

**Comment by pastk** [comment]
> 
> I am sorry but i cannot find the (lv) lang in strings.txt can you please tag the line or tell me the correct abbreviation used in strings.txt file.

Latvian (`lv`) was merged recently https://github.com/organicmaps/organicmaps/pull/9211
Please update your fork and rebase.

File: android/app/src/main/res/values-eu/strings.xml (hunk 2 above)
Comment (alexgabi) [reply android/app/src/main/res/values-eu/strings.xml:863]: 859    <string name="track_recording_toast_nothing_to_save">Arrastoaren grabazioa hutsik dago; ez dago ezer gordetzeko</string>
863      <string name="edit_track">Arrastoa editatu</string>

**Review by vng — APPROVED** [review]

File: data/strings/strings.txt (hunk 4 above)
Comment (patepelo) [reply data/strings/strings.txt:32378]: ES is good!

Comment (patepelo) [reply data/strings/strings.txt:32378]: ES: maybe "Elegir color" is more concise.

File: data/strings/strings.txt [hunk 5]
```diff
@@ -32346,10 +32346,94 @@
...
+    ar = تعديل المسار
+    az = Marşrutu redaktə edin
+    be = Рэдагаваць маршрут
+    bg = Редактиране на маршрут
+    ca = Edita el recorregut
+    cs = Upravit trasu
+    da = Rediger ruten
+    de = Route bearbeiten
+    el = Επεξεργασία διαδρομής
+    es = Editar trajeto
```
Comment (patepelo) [inline data/strings/strings.txt:32405]: es = Editar traza

File: data/strings/strings.txt [hunk 6]
```diff
@@ -32346,10 +32346,94 @@
...
+    it = Modifica il percorso
+    ja = ルートを編集
+    ko = 경로 편집
+    lt = Redaguoti maršrutą
+    lv = Ceļa labošana
+    mr = मार्ग संपादित करा
+    nb = Rediger ruten
+    nl = Route bewerken
+    pl = Edytuj trasę
+    pt = Editar rota
```
Comment (matheusgomesms) [inline data/strings/strings.txt:32424]:
```suggestion
    pt = Editar trajeto
```

File: android/app/src/main/res/values-es/strings.xml [hunk 7]
```diff
@@ -864,6 +864,8 @@
     <string name="track_recording_toast_nothing_to_save">La ruta está vacía - no hay nada que guardar</string>
     <!-- Error message when there are no File Manager apps installed to select a folder when importing Bookmarks and Tracks -->
     <string name="error_no_file_manager_app">No se puede mostrar el diálogo de selección de carpetas porque no hay ninguna app adecuada instalada en su dispositivo. Instale una aplicación de gestión de archivos e inténtelo de nuevo</string>
+  <string name="choose_color">Elegir el color</string>
+  <string name="edit_track">Editar trajeto</string>
```
Comment (patepelo) [inline android/app/src/main/res/values-es/strings.xml:868]: ES: 
name="choose_color" > Elegir color
name="edit_track" > Editar traza

**Review by patepelo — CHANGES_REQUESTED** [review]
ES:
name="choose_color" > Elegir color
name="edit_track" > Editar traza

File: data/strings/strings.txt [hunk 8]
```diff
@@ -32346,10 +32346,94 @@
...
     en = Edit Track
+    af = Wysig Roete
+    ar = تعديل المسار
+    az = Marşrutu redaktə edin
+    be = Рэдагаваць маршрут
+    bg = Редактиране на маршрут
+    ca = Edita el recorregut
+    cs = Upravit trasu
+    da = Rediger ruten
+    de = Route bearbeiten
```
Comment (map-per) [inline data/strings/strings.txt:32403]:
```suggestion
    de = Track bearbeiten
```

File: data/strings/strings.txt [hunk 9]
```diff
@@ -32346,10 +32346,94 @@
...
     en = Choose Color
+    af = Kies Kleur
+    ar = اختر اللون
+    az = Rəngi seçin
+    be = Абярыце колер
+    bg = Изберете цвят
+    ca = Tria el color
+    cs = Vyberte barvu
+    da = Vælg farve
+    de = Farbe wählen
```
Comment (map-per) [inline data/strings/strings.txt:32357]:
```suggestion
    de = Farbe auswählen
```

File: android/app/src/main/res/values-fr/strings.xml [hunk 10]
```diff
@@ -868,6 +868,8 @@
     <string name="track_recording_toast_nothing_to_save">L\'itinéraire est vide - il n\'y a rien à sauvegarder</string>
     <!-- Error message when there are no File Manager apps installed to select a folder when importing Bookmarks and Tracks -->
     <string name="error_no_file_manager_app">Impossible d\'afficher la boîte de dialogue de sélection des dossiers car aucune application appropriée n\'est installée sur votre appareil. Veuillez installer une application de gestion de fichiers et réessayer</string>
+  <string name="choose_color">Choisir la couleur</string>
+  <string name="edit_track">Modifier le trace</string>
```
Comment (Arthur-GYT) [inline android/app/src/main/res/values-fr/strings.xml:872]:
```suggestion
  <string name="edit_track">Modifier le tracé</string>
```

**Review by Arthur-GYT — CHANGES_REQUESTED** [review]

**Review by jimcarst — COMMENTED** [review]
nl ok

**Comment by vng** [comment]
Updated strings for the upcoming release.

**Pull request merged**